macros/scaffold.pl

```diff
@@ -321,7 +324,7 @@ sub End {
   Scaffold->Error("Scaffold ended with section was still open") if $self->{current_section};
   my $self = $scaffold;
   push(@{$self->{output}},splice(@$PG_OUTPUT,0));                      # collect any final non-section output
-  $self->open_sections(@{$self->{open}});                              # make the open sections be displayed
+  $self->hide_other_results(@{$self->{open}});                         # hide results of unnopened sections in the results table
```

Comment:
```suggestion
  $self->hide_other_results(@{$self->{open}});                         # hide results of unopened sections in the results table
```

Reply:
Trying Andrew's GitHub tip.

Reply:
I have a two-part exercise (three answer blanks, then two). If I enter some answers in the first three blanks, even one of them incorrect, then when I click submit, the results table still has five rows. The bottom two rows are grayed out. This is all while logged in as a student. Is this what I should be seeing? The wording here made me think I should only see three rows.

<img width="864" alt="Screen Shot 2021-05-13 at 2 42 27 PM" src="https://user-images.githubusercontent.com/4732672/118191500-871d3900-b3f9-11eb-99bb-c5862a6bb271.png">